src/Database/test.js

```diff
@@ -4,619 +4,651 @@ import { noop } from '../utils/fp'
 import { CollectionChangeTypes } from '../Collection/common'
 import * as Q from '../QueryDescription'
 
-describe('unsafeResetDatabase', () => {
-  it('can reset database', async () => {
-    const { database, tasks } = mockDatabase({ actionsEnabled: true })
+describe('Database', () => {
+  it(`implements get()`, () => {
```

Comment:
sorry about the massive diff- github's not smart enough to see it's just indentation. this test (next few lines) is the only thing I changed